common/src/androidTest/java/com/microsoft/identity/common/HttpRequestTest.java

```diff
@@ -79,265 +77,137 @@ public void testNullRequestUrl() throws IOException {
     }
 
     /**
-     * Verify that HTTP GET succeed and no retry happens.
+     * Verify that HTTP GET succeed via HttpResponse.sendGet().
```

Comment:
>via HttpResponse.sendGet(). [](http://example.com/codeflow?start=36&length=27)

We can add this part to testmethod name..
testHttpGetSucceedViaHttpResponseSendGet() ..

Reply:
same at other places too, lets make the names do the documentation as much as possible